hazelcast/src/test/java/com/hazelcast/config/XmlConfigSchemaLocationTest.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.config;
+
+import com.hazelcast.nio.IOUtil;
+import com.hazelcast.test.HazelcastSerialClassRunner;
+import com.hazelcast.test.HazelcastTestSupport;
+import com.hazelcast.test.annotation.SlowTest;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.reflections.Reflections;
+import org.reflections.scanners.ResourcesScanner;
+import org.w3c.dom.Attr;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import static com.hazelcast.test.TestCollectionUtils.setOf;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(HazelcastSerialClassRunner.class)
+@Category(SlowTest.class)
+public class XmlConfigSchemaLocationTest extends HazelcastTestSupport {
+    private static final Set<String> WHITELIST = setOf("hazelcast-sample-service.xsd");
```

Comment:
I didn't quite get WHITELIST?

Reply:
see this file: https://github.com/jerrinot/hazelcast/blob/4031fca68c08a763151649ca7d41d41713deafdd/hazelcast/src/test/resources/com/hazelcast/config/hazelcast-service.xml#L22

It's an example of a custom Hazelcast service. 

The example defines own schema, but the schema is not available anywhere on the internet. It's just an example after all. So the test has to skip it.

Reply:
Could you add a comment with reference to the `ServiceConfigTest` on the constant?